pkg/api/types.go

```diff
@@ -814,6 +814,10 @@ type PodSpec struct {
 	// NodeSelector is a selector which must be true for the pod to fit on a node
 	NodeSelector map[string]string `json:"nodeSelector,omitempty"`
 
+	// ServiceAccount is the name of the ServiceAccount to use to run this pod
+	// The pod will be allowed to use secrets referenced by the ServiceAccount
```

Comment:
Ultimately, that the ServiceAccount admission plugin will reject pods that reference secrets that are not in the pod's service account's list of secrets. Removing from here, and adding to the admission plugin doc, since that's what accomplishes that behavior